### ISSUE TITLE
Change log level

### ISSUE THREAD

**Issue by Bananeweizen** [societe-generale/arch-unit-maven-plugin#10]
Currently every analyzed class is logged via INFO log level, looking like this:
```
[INFO] --- arch-unit-maven-plugin:1.0.2:arch-test (default) @ com.company.foobar ---
[INFO] Analysing class 'com/foo/class1'
[INFO] Analysing class 'com/foo/class2'
[INFO] Analysing class 'com/foo/class3'
[INFO] Analysing class 'com/foo/class4'
```
That is way to much, since INFO is the standard maven console log level. Please change that to DEBUG or similar. Other maven plugins do not output anything in case of working correctly or just the number of affected artifacts.

**Comment by vincent-fuchs** [comment]
you're right, we should change it to DEBUG. This is left from our exploration days ;-)

**Comment by vincent-fuchs** [comment]
OK, so after a quick look in the code base, the problem is that's not coming from the Maven plugin, but from ArchUnit lib itself, see https://github.com/TNG/ArchUnit/blob/5a26ce04fa3f8df3f4c3cb07d1fa675ce557da20/archunit/src/main/java/com/tngtech/archunit/core/importer/JavaClassProcessor.java#L101 

So not sure how we can workaround this.. probably by placing a logback config in the plugin and override the level.. I'll try and keep you posted

**Comment by Bananeweizen** [comment]
To me it looks more like an unintended level, since everything else in that class already uses debug level for logging. Let's see if my PR is accepted.

**Comment by vincent-fuchs** [comment]
thanks for the PR on ArchUnit lib - will wait for the status change of https://github.com/TNG/ArchUnit/pull/180 and subsequent release. Once we upgrade, issue will be solved.

**Comment by codecholeric** [comment]
Should be fixed with the next release: https://github.com/TNG/ArchUnit/commit/2ac6b981a3ca5ae845dcaa25709988034c6426ed

**Comment by vincent-fuchs** [comment]
fixed in 2.0.0 with https://github.com/societe-generale/arch-unit-maven-plugin/commit/1b1582ecd763adca2b5fded9ffbfef5ca3733179
